example/example.djinni

```diff
@@ -1,6 +1,6 @@
 item_list = record {
     items: list<string>;
-}
+} deriving(parcelable)
```

Comment:
Why is this needed in the example?  Seems unnecessary, and the example should stay simple and understandable.
Given the generated code isn't updated either, I suspect maybe this was a test case you didn't intend to push.